packages/office-ui-fabric-react/src/components/pickers/PeoplePicker/PeoplePickerItems/SelectedItemDefault.tsx

```diff
@@ -51,12 +52,15 @@ export const SelectedItemDefault: (props: IPeoplePickerItemProps) => JSX.Element
           size={ PersonaSize.size28 }
         />
       </div>
-      <IconButton
-        onClick={ onClickIconButton(onRemoveItem) }
-        iconProps={ { iconName: 'Cancel', style: { fontSize: '12px' } } }
-        className={ css('ms-PickerItem-removeButton', styles.removeButton) }
-        ariaLabel={ removeButtonAriaLabel }
-      />
+      {
+        disabled ? null :
```

Comment:
If the disabled property is added then it should impact more than just the IconButton, what about the styles of the rest of the item?

Reply:
The disable property wasn't being referenced in the first place, so I'm not sure how this will affect the rest of them. Similar to a textbox, if it's "disabled" then you shouldn't be able to add/remove text. This is similar where the personas shouldn't show the "x" icon when it's "disabled".

Reply:
I think that we might want to add styling if the item is disabled, this is a bigger question than just disabling the X. If the item is disabled should it be selectable? Should it even appear in the list?

Reply:
@gunjandatta We're trying to understand your use case better. Can you elaborate on what you're trying to do?

Reply:
It seems this PR got stale. Sorry about that.

At a minimum, we need a change file. You should run `npm run change` from the root to describe your change and add the json file it produces.

`disabled` implies a state for a button; it is there, it is visible, but greyed out. Perhaps `unremovable` or `readonly` flag would make more sense?

Reply:
I think just adding styling to the item, in addition to removing the x is good. It's just that the item should be greyed out and actually appear disabled as well.